edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/GradebookServiceHibernateImpl.java

```diff
@@ -1228,6 +1228,57 @@ public Object doInHibernate(Session session) throws HibernateException {
 	  	return (List)getHibernateTemplate().execute(hc);
 	  }
 
+	/**
+	 * Gets all AssignmentGradeRecords on the gradableObjectIds limited to students specified by studentUids
+	 */
+	private List<AssignmentGradeRecord> getAllAssignmentGradeRecordsForGbItems(final List<Long> gradableObjectIds, final List studentUids)
+	{
+		HibernateCallback hc = new HibernateCallback()
+		{
+			public Object doInHibernate(Session session) throws HibernateException
+			{
+				List<AssignmentGradeRecord> gradeRecords = new ArrayList<AssignmentGradeRecord>();
```

Comment:
Would have used diamond operators, but for omitted them for ease of reverse merging